core/indigo-core/molecule/molecule_json_loader.h

```diff
@@ -34,26 +34,30 @@
 #pragma warning(disable : 4251)
 #endif
 
-namespace indigo
+class RGroupDescriptionList : public std::list<std::pair<int, std::reference_wrapper<rapidjson::Value>>>
 {
+};
```

Comment:
Hmm, why not simply using?
It makes sense to wrap it into `namespace Indigo`.
(all optional = now now)

Reply:
Fixed